integration_tests/mockApis/educationAndWorkPlanApi.ts

```diff
@@ -893,6 +893,32 @@ const stubGetInductionShortQuestionSet = (prisonNumber = 'G6115VJ'): SuperAgentR
           trainingTypes: ['FULL_UK_DRIVING_LICENCE'],
           trainingTypeOther: null,
         },
+        previousWorkExperiences: {
```

Comment:
Add previous work experience to the wiremock stub for retrieving a Short question set induction from the API